pom.xml

```diff
@@ -147,16 +147,16 @@
     <module>bom</module>
     <module>broker</module>
     <module>core</module>
+    <module>service-base</module>
```

Comment:
changing the order in the pom does not have any influence on the order in which the projects are built AFAIK. The build order is determined by means of resolving dependencies between the modules at the beginning of the (reactor) build. These modules will be built before `example` simply because `example` depends on them (either directly or transitively) ...